src/test/java/teammates/test/pageobjects/InstructorCourseEditPage.java

```diff
@@ -121,6 +121,24 @@ public boolean clickEditInstructorLink(int instrNum) {
         return isEditable;
     }
     
+    public void clickCancelEditInstructorLink(int instrNum) {
+        getCancelEditInstructorLink(instrNum).click();
+    }
+    
+    public boolean isInstructorFormNotEditable(int instrNum) {
```

Comment:
Returning a `notEditable` boolean seems to go against the convention of booleans, but I guess this is due to implementation, as using `waitForElementVisibility` instead may result in false positives.